examples/example_for_updating_metadata.py

```diff
@@ -0,0 +1,115 @@
+from sparc_me import Dataset
+
+if __name__ == '__main__':
+    save_dir = "./tmp/template/"
+
+    dataset = Dataset()
+
+    # Step:1 list categories and dataset_description elements
+    categories = dataset.list_categories(version="2.0.0")
+    elements = dataset.list_elements(category="dataset_description", version="2.0.0")
+    # list code_parameters elements
+    # elements = dataset.list_elements(category="code_parameters", version="2.0.0")
+
+    # Step2, way1: load dataset from template
+    dataset.load_from_template(version="2.0.0")
+    #
+    # Save the template dataset.
+    # dataset.save(save_dir=save_dir)
+
+    # Step2, way2： load dataset from existing dataset
+    # dataset.load_dataset(dataset_path=save_dir)
+
+    # Step3, get dataset_description, code_description, code_parameters metadataEditor
+    dataset_description_editor = dataset.get_metadata_editor(category="dataset_description")
```

Comment:
rename get_metadata_editor to get_metadata

Reply:
fixed by [2394cf5](https://github.com/SPARC-FAIR-Codeathon/sparc-me/pull/92/commits/2394cf5c2aa697036e27338520944c4a76ebc182)